kiteconnect/connect.py

```diff
@@ -81,6 +81,10 @@ class KiteConnect(object):
     STATUS_REJECTED = "REJECTED"
     STATUS_CANCELLED = "CANCELLED"
 
+    # GTT order type
+    GTT_OCO = "two-leg"
```

Comment:
I think its better to name it `GTT_TWO_LEG` since we call it two leg everywhere.